plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/plan/CubridPlanNode.java

```diff
@@ -33,11 +33,14 @@ public class CubridPlanNode extends AbstractExecutionPlanNode
     private static final String COST = "cost";
     private static final String CLASS = "class";
     private static Map<String, String> classNode = new HashMap<>();
+    private static Map<String, String> terms = new HashMap<>();
     private String fullText;
     private String nodeName;
     private String name;
-    private Number cost;
-    private Number row;
+    private String index;
+    private String term;
+    private Long cost;
+    private Long row;
```

Comment:
Please make these `long` and remove associated `@NotNull` annotations. This annotation is not required for primitives.